doc/whats_new/v0.22.rst

```diff
@@ -60,6 +60,20 @@ Changelog
 - |Fix| :func:`utils.check_array` now correctly converts pandas DataFrame with
   boolean columns to floats. :pr:`15797` by `Thomas Fan`_.
 
+- |Fix| :func:`utils.check_is_fifted` accepts back an explicit ``attributes``
+  argument to check for specific attributes as explicit markers of a fitted
+  estimator. When no explicit attributes are provided, only the attributes
+  ending with a single "_" are used as "fitted" markers. This change is made to
+  restore some backward compatibility with the behavior of this utility in
+  version 0.21. :pr:`15947` by `Thomas Fan`_.
+
+:mod:`sklearn.inspection`
```

Comment:
I think this something due to a merge conflict but you added back a section.

Reply:
Indeed I did not read carefully, sorry.